Update configure.md (#510)
percona/pmm-doc: docs/how-to/configure.md

The first telemetry reporting of a new PMM Server instance is delayed by 24 hours to allow sufficient time to disable the service for those that do not wish to share any information.

> There is a landing page for this service, available at [check.percona.com](https://check.percona.com), which clearly explains what this service is.

Reviewer: ```suggestion
The landing page for this service, [check.percona.com](https://check.percona.com), explains what this service is.
```
